### ISSUE TITLE
Python library not found

### ISSUE THREAD

**Issue by duryjames** [nvbn/rhythmbox-gmusic#44]
Hi,

I am having the following issue:

http://postimg.org/image/5laztkra9/

installed via pip, the "From the Shell" example:

```
sudo apt-get install python-pip git-core
sudo pip install git+https://github.com/nvbn/rhythmbox-gmusic.git#egg=rhythmbox-gmusic
```

My python version is Python 2.7.6

**Comment by gerbal** [comment]
Rhythmbox version 3 switched to python3. Rhythmbox-gmusic won't work with python3 currently

**Comment by gerbal** [comment]
Duplicate of #39

**Comment by p0lar-bear** [comment]
Just an FYI, this runs deeper than rhythmbox-gmusic not being written for python3. gmusicapi is python2 only because at least one of the libraries it depends on (mutagen, namely, but there's likely more) doesn't have a python3-compatible version.

Until someone is able to port gmusicapi to python3, the easiest solution for Ubuntu users is to avoid upgrading to 14.04 Trusty for now, as Rhythmbox 2.99 is only available for 13.10 Saucy.

Edit: Opened an issue over at simon-weber/Unofficial-Google-Music-API#247, waiting on a reply.
